src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private static final Random RANDOM = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```

I think we could make this field not static